app/controllers/api/databases.php

```diff
@@ -1669,16 +1672,55 @@ function updateAttribute(
 
         $collection = $dbForProject->getDocument('database_' . $database->getInternalId(), $collectionId);
 
+
         if ($collection->isEmpty()) {
             throw new Exception(Exception::COLLECTION_NOT_FOUND);
         }
 
-        $attributes = $collection->getAttribute('attributes');
+        $queries = Query::parseQueries($queries);
+        foreach ($queries as $query) {
+            if ($query->getMethod() ===  Query::TYPE_SELECT) {
+                throw new Exception(Exception::GENERAL_QUERY_INVALID, "Select queries are not valid.");
+            }
+        }
+        \array_push($queries, Query::equal('collectionId', [$collectionId]), Query::equal('databaseId', [$databaseId]));
+
 
-        $response->dynamic(new Document([
-            'total' => \count($attributes),
-            'attributes' => $attributes
-        ]), Response::MODEL_ATTRIBUTE_LIST);
+         // Get cursor document if there was a cursor query
+         $cursor = Query::getByType($queries, Query::TYPE_CURSORAFTER, Query::TYPE_CURSORBEFORE);
+         $cursor = reset($cursor);
+
+        if ($cursor) {
+            $attributeId = $cursor->getValue();
+
+            $cursorDocument = Authorization::skip(fn() => $dbForProject->find('attributes', [Query::equal('collectionId', [$collectionId]), Query::equal('databaseId', [$databaseId]), Query::equal('key', [$attributeId]), Query::limit(1)]));
```

Comment:
Let's add a line breaks for readability:

```suggestion
            $cursorDocument = Authorization::skip(fn() => $dbForProject->find('attributes', [
                Query::equal('collectionId', [$collectionId]),
                Query::equal('databaseId', [$databaseId]),
                Query::equal('key', [$attributeId]), 
                Query::limit(1),
            ]));
```

Reply:
done this